src/types/keywords.types.ts

```diff
@@ -0,0 +1,7 @@
+export interface Keywords {
+  label: {
+    nb?: string;
+    em?: string;
```

Comment:
"em"? skal være "en"? kan det være andre språk her også, feks nynorsk eller samisk (eller hva som helst)?
```suggestion
    en?: string;
```

Reply:
hvis det er en typo her, er det rart det ikke har dukket opp noen feilmeldinger der hvor du bruker `.en` nå. tyder kanskje på at typen ikke brukes der den skal?

Reply:
disse brukes kanskje bare for `getTranslationString()`? isf gir det mening at det ikke kommer noen feilmeldinger

Reply:
Jepp, det var en typo. Er kun brukt i `getLanguageString`